### PR TITLE
Exec 2022 01 14

### PR DESCRIPTION
**Pull request by mkrorbrkr**
@LongForWisdom Exec copy for this week

### CODE REVIEW

File: governance/votes/Executive vote - January 14, 2022.md [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+The Governance Facilitator(s) and the Protocol Engineering Core Unit have placed an executive proposal into the voting system. MKR Holders should vote for this proposal if they support the following alterations to the Maker Protocol.
+
+If you are new to voting in the Maker Protocol, please see the [voting guide](https://community-development.makerdao.com/en/learn/governance/how-voting-works/) to learn how voting works, and this [wallet setup guide](https://community-development.makerdao.com/en/learn/governance/voting-setup/) to set up your wallet to vote.
+
+---
+
+## Executive Summary
+
+If this executive proposal passes, the following **changes** will occur within the Maker Protocol:
+- 10,000,000 DAI will be transferred from the Optimisim Escrow to a user who accidentally burned DAI on the Optimism chain.
```
Comment (LongForWisdom) [inline governance/votes/Executive vote - January 14, 2022.md:19]: I tend to write anything over 999,999 as x million DAI. I think it's easier to parse than the set of zeros.

File: governance/votes/Executive vote - January 14, 2022.md [hunk 2]
```diff
@@ -0,0 +1,88 @@
+---
+title: Template - [Executive Vote] Recover DAI from Optimism Escrow, Changes to Uniswap LP Vault Debt Floor Parameters, Delegate Compensation - January 14, 2022
+summary: Recover DAI from Optimism escrow, Uniswap LP Vault Debt Floor parameter changes, Recognized Delegate compensation distribution for December.
```
Comment (LongForWisdom) [inline governance/votes/Executive vote - January 14, 2022.md:3]: Maybe do Debt Floor (dust) we're still in the process of pushing people towards other terminology.

File: governance/votes/Executive vote - January 14, 2022.md (hunk 1 above)
Comment (LongForWisdom) [reply governance/votes/Executive vote - January 14, 2022.md:19]: Also maybe be consistent on linking to polls / forum posts in the summary. You did for the comp trial, but not for this or the dust thing?

File: governance/votes/Executive vote - January 14, 2022.md [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+
+If this executive proposal does not pass within 30 days, then it will expire and can no longer have any effect on the Maker Protocol.
+
+---
+
+## Proposal Details
+
+### Recover DAI from Optimism Escrow
+
+As per this successful [governance poll](https://vote.makerdao.com/polling/Qmcfb72e), **10,000,000 DAI** will be transferred to forum user lost_some_dai at the address [0xc9b48B787141595156d9a7aca4BC7De1Ca7b5eF6](https://etherscan.io/address/0xc9b48B787141595156d9a7aca4BC7De1Ca7b5eF6) if this executive proposal passes.
```
Comment (LongForWisdom) [inline governance/votes/Executive vote - January 14, 2022.md:35]: 10,000,000 DAI -> 10 million DAI

Comment (LongForWisdom) [reply governance/votes/Executive vote - January 14, 2022.md:35]: Worth some @, quotations or emphasis on the username? Link to forum account? etc?

File: governance/votes/Executive vote - January 14, 2022.md [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+
+For further information, please see the initial Signal Request on the MakerDAO forum [here](https://forum.makerdao.com/t/signal-request-should-makerdao-assist-in-recovering-dai-locked-on-optimism-escrow/12307).
+
+### Uniswap LP Vault [Debt Floor (`dust`)](https://manual.makerdao.com/parameter-index/vault-risk/param-debt-floor) Increases
+
+As per this successful [governance poll](https://vote.makerdao.com/polling/QmUSfhmF), the following Uniswap LP Vault [Debt Floor (`dust`)](https://manual.makerdao.com/parameter-index/vault-risk/param-debt-floor) changes will be made to the Maker Protocol if this executive proposal passes.
+
+**UNIV2DAIETH-A**
```
Comment (LongForWisdom) [inline governance/votes/Executive vote - January 14, 2022.md:43]: I would be tempted to just inline the bolded titles here. IE: Increase the **UNIV2DAIETH-A** Debt Floor by...

**Review by LongForWisdom — CHANGES_REQUESTED** [review]
Good generally, see comments.

File: governance/votes/Executive vote - January 14, 2022.md (hunk 3 above)
Comment (mkrorbrkr) [reply governance/votes/Executive vote - January 14, 2022.md:35]: Changed both and added comment by PE about how changes DAI transfer will happen

File: governance/votes/Executive vote - January 14, 2022.md (hunk 2 above)
Comment (mkrorbrkr) [reply governance/votes/Executive vote - January 14, 2022.md:3]: changed in summary, did you want the title changed as well?

File: governance/votes/Executive vote - January 14, 2022.md (hunk 1 above)
Comment (mkrorbrkr) [reply governance/votes/Executive vote - January 14, 2022.md:19]: Fixed both

File: governance/votes/Executive vote - January 14, 2022.md (hunk 2 above)
Comment (LongForWisdom) [reply governance/votes/Executive vote - January 14, 2022.md]: Title is fine, I think.

**Review by LongForWisdom — APPROVED** [review]

**Pull request merged**